### PR TITLE
Fix some tests

### PR DESCRIPTION
**Pull request by alexmiddeleer**
Going to see if I can fix Travis.

### CODE REVIEW

File: postgres/init.sql [hunk 1]
```diff
@@ -308,19 +308,19 @@ create function mythgard.search_decks(deckName varchar(255), authorName varchar(
     intersect
 
     -- cards filter
-    SELECT deck.* from mythgard.deck left join mythgard.card_deck on (deck.id = card_deck.deck_id) left join mythgard.card on (card.id = card_deck.card_id) left join mythgard.card_faction on (card.id = card_faction.card_id) left join mythgard.faction on (faction.id = card_faction.faction_id)
+    SELECT deck.* from mythgard.deck left join mythgard.card_deck on (deck.id = card_deck.deck_id) left join mythgard.card on (card.id = card_deck.card_id) left join mythgard.card_faction on (card.id = card_faction.card_id)
```
Comment (alexmiddeleer) [inline postgres/init.sql]: these extra joins were just cruft

File: e2e/cypress/integration/cards-page.js [hunk 2]
```diff
@@ -146,12 +152,16 @@ describe('Cards Page', function() {
         return getPagingTotalAsInt();
       })
       .then(length => {
-        expect(allCardsLength).to.be.above(length);
+        expect(allCardsLength).to.be.above(
+          length,
+          'setup for clear button test'
+        );
         cy.get('[data-cy="cardSearchClear"]').click();
+        cy.wait(500);
```
Comment (alexmiddeleer) [inline e2e/cypress/integration/cards-page.js]: I can't think of anything else to do here, sadly. I don't think a network request should happen b/c of the cache, so can't tell cy to wait for that. There's probably a better thing to do here as cypress is pretty clever but I'm going to have to research that later. LMK if you can think of anything.

**Comment by lily-pytel** [comment]
Haven't seen that green checkmark in a thousand years

**Comment by alexmiddeleer** [comment]
I know it. it feels... good!

**Pull request merged**